heidrun/TN_mods.rb

```diff
@@ -0,0 +1,144 @@
+Krikri::Mapper.define(:tn_mods, :parser => Krikri::ModsParser) do
+  provider :class => DPLA::MAP::Agent do
+    uri 'http://dp.la/api/contributor/tn'
+    label 'Tennessee Digital Library'
+  end
+
+  dataProvider :class => DPLA::MAP::Agent do
+    providedLabel record.field('mods:recordInfo', 'mods:recordContentSource')
+  end
+
+  isShownAt :class => DPLA::MAP::WebResource do
+    uri record.field('mods:location', 'mods:url')
+         .match_attribute(:usage, 'primary display')
```

Comment:
Should this be `:usage, 'primary'`, as seen at: https://gist.github.com/no-reply/458a8d09e8d622c831a7#file-record-xml-L44 ?